net.sf.eclipsecs.ui/META-INF/MANIFEST.MF

```diff
@@ -23,7 +23,6 @@ Import-Package: org.eclipse.core.commands,
  org.eclipse.core.runtime,
  org.eclipse.core.runtime.jobs,
  org.eclipse.core.runtime.preferences,
- org.eclipse.e4.ui.model.application;resolution:=optional,
```

Comment:
I think this dependency is still needed, as a few Eclipse3/4 switches depend on it, see https://github.com/checkstyle/eclipse-cs/blob/7eb81a2068e12ec0d9accee30db2ea8dbec3de49/net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/CheckstyleUIPlugin.java#L135

This could be refactored of course if Eclipse 3 compatibility is dropped altogether.

Reply:
Ah, good advice. In that case, Indigo is not sufficient. I had taken that since the feature.xml required at least eclipse.core 3.7. I will check the class name references more closely and come up with a modified PR that includes a slightly higher Eclipse version as target. Thanks.